x-pack/plugins/security_solution/public/common/components/exceptions/use_fetch_or_create_rule_exception_list.test.tsx

```diff
@@ -168,6 +168,24 @@ describe('useFetchOrCreateRuleExceptionList', () => {
         expect(patchRule).toHaveBeenCalledTimes(1);
       });
     });
+    it('invokes onSuccess indicating that the rule changed', async () => {
+      const onSuccess = jest.fn();
+      await act(async () => {
+        const { waitForNextUpdate } = renderHook(() =>
```

Comment:
You could also add `onSuccess` to the `render` function here https://github.com/elastic/kibana/blob/1dfc3654c838730c89ac386d673d7b472b5279fe/x-pack/plugins/security_solution/public/common/components/exceptions/use_fetch_or_create_rule_exception_list.test.tsx#L96, which kind of abstracts out the `renderHook` call.

Reply:
Yeah, this was a quick sanity-check test; I'll refactor to use the test helper.